app/assets/javascripts/pretty-text/addon/mentions.js

```diff
@@ -0,0 +1,31 @@
+export function mentionRegex(unicodeUsernames) {
```

Comment:
what is the reason of moving this file btw? it was not accessible otherwise?

Reply:
Yes, I moved so I could use it outside the rule.